### ISSUE TITLE
Error Stream xxx could not be initialized

### ISSUE THREAD

**Issue by austre** [gantry/gantry5#2984]
Joomla 3.10.14
PHP 7.4

Unfortunely, from night to morning my website shows message below:

Error: Stream '\Rockettheme\Toolbox\StreamWrapper\Stream' could not be initialized.: Stream '\Rockettheme\Toolbox\StreamWrapper\Stream' could not be initialized.

https://sgalite.cref16.org.br/
![image](https://user-images.githubusercontent.com/4764413/149765801-a4e17d34-5191-4e2b-bee7-c798dfcb641e.png)

**Comment by mahagr** [comment]
@austre Can you please try Joomla CI build in https://gantry.org/downloads#ci-builds (give it 1-2 hours to build), it should have better error message which may help to point out what is going wrong.

**Comment by austre** [comment]
@mahagr thanks for reply. I'll take a look about CI Build today.

**Comment by mahagr** [comment]
@austre Let me know what the new error says, it would be nice to have this issue figured out. I think it may be related to file permissions somehow.

**Comment by austre** [comment]
Hi @mahagr

Sorry for the delay. I was testing several alternatives to understand and solve the problem. With the help of hosting support we discovered that there was a mismatch in a snuffleupagus rule.

I don't know this snuffleupagus well, but I understood that it would be some security service they use and it classified that file wrong.

Hosting support tweaked the snuffleupagus rules, we restored a backup and the application was working correctly.
